certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/certificateOutput.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.certifikit.cli
+
+import app.cash.certifikit.BasicConstraints
+import app.cash.certifikit.Certificate
+import app.cash.certifikit.ObjectIdentifiers
+import java.security.cert.X509Certificate
+import java.time.Instant.ofEpochMilli
+import javax.net.ssl.X509TrustManager
+import okhttp3.internal.platform.Platform
+import okio.ByteString
+import okio.ByteString.Companion.toByteString
+import picocli.CommandLine.Help.Ansi
+
+fun X509Certificate.sha256Hash(): ByteString =
+  publicKey.encoded.toByteString()
```

Comment:
I think this method is misnamed. The hash of the certificate and the hash of the public key are different hashes, and serve different purposes. 

(I’m thinking about the hash displayed in the browser’s certificate UI)